inputs/uprev_rdg_storage_format_version.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#
+# Run from build directory to uprev the storage format version of the input rdgs
+usage="./uprev_rdg_storage_format_version.sh <storage_format_version>"
+example="./uprev_rdg_storage_format_version.sh 3"
+
+if [ -z "$1" ]; then
+    echo "Must specify new storage format version" >&2
+    echo "usage: $usage"
+    echo "example: $example"
+    exit 1
+fi
+
+set -eu -o pipefail
+
+if [[ ! -f CMakeCache.txt ]]; then
+    echo run from build directory >&2
+    exit 1
+fi
+
+NEW_VERSION=$1
+
+NEW_VERSION_DIR=storage_format_version_$NEW_VERSION/
```

Comment:
That is a long directory name.  Should I just get over it?

Reply:
I'm a fan of verbosity and clarity in organization. I could see an argument for just using "v#" but RDGs themselves already have versions so I think that would be confusing.